### PR TITLE
Upgrade to new namespace convention. Add py 3.11 and py3.12

### PR DESCRIPTION
**Pull request by fredboudon**

### CODE REVIEW

File: setup.py
Comment (pradal) [inline setup.py:1]: It seems that the namespace has not been managed

Comment (pradal) [reply setup.py:1]: namespace_packages = [namespace],
        create_namespaces = False,

has to be removed line 71

Comment (pradal) [reply setup.py:1]: and namespace has to be removed line 87

**Review by pradal — CHANGES_REQUESTED** [review]
Just update the setup.py

**Comment by pradal** [comment]
@fredboudon : we need PyQGLViewer in openalea3 channel...

> pyqglviewer >=1.3

**Comment by fredboudon** [comment]
You can use the one that is built on my channel. I'm not sure to know how to copy these packages

**Comment by pradal** [comment]
I have make a fork on OpenAlea to have it in the OpenAlea channel also.
Then the openalea3 channel is self-consistent.
Thanks again.

**Pull request merged**